### PR TITLE
Feature: adds cronjob to periodically clear requests_cache file and shrink db

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: salt/lax/bot-lax-adaptor.sls [hunk 1]
```diff
@@ -194,3 +197,11 @@ uwsgi-bot-lax-adaptor:
 #        - require:
 #            - uwsgi-bot-lax-adaptor
 
+
+periodically-remove-expired-cache-entries:
+    cron.present:
+        - user: {{ pillar.elife.deploy_user.username }}
+        - name: cd /opt/bot-lax-adaptor/ && ./clear-expired-requests-cache.sh
+        - identifier: rm-expired-cache-entries
+        - special: "@weekly"
```
Comment (giorgiosironi) [inline salt/lax/bot-lax-adaptor.sls]: could we pin a particular day?

Comment (giorgiosironi) [reply salt/lax/bot-lax-adaptor.sls]: IIRC this would be only on `lax--*--1` and not on node 2 of an environment, would cross check that

Comment (lsh-0) [reply salt/lax/bot-lax-adaptor.sls]: > could we pin a particular day?

Of course we can pin on a particular day. Which day would you like? I'll go with Monday ;)

Comment (lsh-0) [reply salt/lax/bot-lax-adaptor.sls]: fyi, encountered this bug switching away from "@special": https://github.com/saltstack/salt/issues/44530

**Pull request merged**